src/domain/entities/MasterLogFrame.ts

```diff
@@ -0,0 +1,9 @@
+import { IndicatorType } from "$/domain/entities/Indicator";
+import { Id, Ref } from "$/domain/entities/Ref";
+
+export type MasterLogFrame = {
```

Comment:
What's a Master Log Frame? Maybe we can add some TSDocs here, explaining its role in the app.